### ISSUE TITLE
ipsec services gets registered with external DNS services

### ISSUE THREAD

**Issue by galal-hussein** [rancher/rancher#6790]
**Rancher Version:**
v1.2.0-pre4-rc15
**Docker Version:**
1.12
**OS and where are the hosts located? (cloud, bare metal, etc):**
Ubuntu 16.04
**Setup Details: (single node rancher vs. HA rancher, internal DB vs. external DB)**
single node rancher
**Environment Type: (Cattle/Kubernetes/Swarm/Mesos)**
Cattle
**Steps to Reproduce:**
- Deploy any of the external DNS service catalogs.

**Results:** 
You will notice that the ipsec infrastructure service will get registered with the external dns service:
```
11/28/2016 3:30:11 AMtime="2016-11-28T01:30:11Z" level=info msg="Adding dns record: {ipsec.ipsec.default.x.x. [y.y.y.y z.z.z.z] A 120}"
```

**Comment by janeczku** [comment]
To validate fix deploy Route53 v0.6.0 from infra catalog and then do a stack upgrade changing the image to "rancher/external-dns:v0.6.1".

**Comment by galal-hussein** [comment]
@janeczku  Validated the fix with Route53 on Rancher v1.2.0-rc2 the dns service didn't register ipsec or any system services.
